packages/contracts/test/userStateTransition.test.ts

```diff
@@ -0,0 +1,428 @@
+// @ts-ignore
+import { ethers as hardhatEthers } from 'hardhat'
+import { ethers } from 'ethers'
```

Comment:
These don't need to be separate variables. `hardhatEthers` is a superset of ethers.